### PR TITLE
WIP - weak sauce skylark integration

### PR DESCRIPTION
**Pull request by mfine**
Simple skylark integration into serial_client and console. Not to be merged.

/cc @mookerji

### CODE REVIEW

File: piksi_tools/console/console.py [hunk 1]
```diff
@@ -333,11 +334,14 @@ def __init__(self):
...
-      if args.reset:
-        link(MsgReset())
-      console = SwiftConsole(link, update=args.update)
-      console.configure_traits()
+      sdriver = SkylarkDriver('d570b9f2-a145-5de5-9134-62b212eb63b9')
+      with sbp.client.Handler(sbp.client.Framer(sdriver.read, None)) as slink:
+        slink.add_callback(serial_link.swriter(link))
+        link.add_callback(logger)
+        if args.reset:
+          link(MsgReset())
```
Comment (mookerji) [inline piksi_tools/console/console.py]: There's a `link.reset()` to use here.

File: piksi_tools/serial_link.py [hunk 2]
```diff
@@ -184,37 +190,40 @@ def main():
...
+          with Handler(Framer(sdriver.read, None)) as slink:
+            slink.add_callback(swriter(link))
+            link.add_callback(printer, SBP_MSG_PRINT_DEP)
+            link.add_callback(log_printer, SBP_MSG_LOG)
+            Forwarder(link, logger).start()
+            Forwarder(link, append_logger).start()
+            # Reset device
+            if args.reset:
+              link(MsgReset())
+            # Setup watchdog
```
Comment (mookerji) [inline piksi_tools/serial_link.py]: Unrelated, but we don't really use the host-side watchdog stuff anymore. Can we just remove it?

**Comment by mookerji** [comment]
Looks good, WIP-wise. What happens if the HTTP driver fails on connection?

File: piksi_tools/console/console.py (hunk 1 above)
Comment (mfine) [inline piksi_tools/console/console.py]: FWIW, there *was* a `link.reset()` - it got removed with the iterator changes.

File: piksi_tools/serial_link.py (hunk 2 above)
Comment (mfine) [inline piksi_tools/serial_link.py]: We should.

**Comment by mookerji** [comment]
ok! I'm closing this and putting up a new version.

**Pull request closed without merging**